pkgs/emacs/wrapper.nix

```diff
@@ -187,4 +219,35 @@ in
         --set EMACSLOADPATH "$siteLisp:"
       fi
     done
+
+    ${lib.optionalString stdenv.hostPlatform.isDarwin ''
```

Comment:
This file is supposed to provide a basic derivation for running Emacs. If possible, I would like you to move this to the home-manager module. Is there any reason we have do do this here?

Reply:
I'd argue that on macOS systems, providing an Emacs.app output _is_ the basic form of an Emacs derivation. On macOS, the canonical means of launching applications is with a `.app` - the new `Emacs.app` output is not specific to home-manager. We provide the home-manager linking for compatibility. So, I'd say that this is the appropriate place for our `Emacs.app` expression to live. Hopefully that makes sense?